### PR TITLE
inlineAction implementation

### PR DESCRIPTION
**Pull request by Surezt22**
Inline Action implemented for Text Input

### CODE REVIEW

File: source/reactnative/src/components/inputs/input.js [hunk 1]
```diff
@@ -57,29 +61,44 @@ export class Input extends React.Component {
 			return null;
 		}
 
+		if (!Utils.isNullOrEmpty(this.inlineAction)) {
+			if (this.inlineAction.type === "Action.ShowCard") {
+				let error = { "error": Enums.ValidationError.ActionTypeNotAllowed, "message": `Inline ShowCard is not supported as of now` };
```
Comment (BalajiR) [inline source/reactnative/src/components/inputs/input.js]: Import `Enums` !!

Comment (BalajiR) [inline source/reactnative/src/components/inputs/input.js]: From Where the `onParseError` is received here ?

Comment (Surezt22) [reply source/reactnative/src/components/inputs/input.js]: onParseError has to be received from InputContextConsumer.  Fixed the issue.

Comment (Surezt22) [reply source/reactnative/src/components/inputs/input.js]: Done

File: source/reactnative/src/components/inputs/input.js [hunk 2]
```diff
@@ -117,10 +136,98 @@ export class Input extends React.Component {
...
+	onClickHandle(onExecuteAction) {
+		if (this.payload.inlineAction.type === Constants.ActionSubmit && !this.isMultiline) {
+			let actionObject = { "type": Constants.ActionSubmit, "data": this.state.text };
+			onExecuteAction(actionObject);
+		}
 	}
 }
 
 const styles = StyleSheet.create({
+	inlineActionText: {
```
[the remark this answers is not in the document]
Comment (Surezt22) [reply source/reactnative/src/components/inputs/input.js]: Done

File: source/reactnative/src/components/inputs/input.js [hunk 3]
```diff
@@ -117,10 +136,98 @@ export class Input extends React.Component {
...
+			</InputContextConsumer>
+		);
+
+	}
+
+	/**
+ 	* @description Invoked on tapping the inline-action image component
+	*/
+	onClickHandle(onExecuteAction) {
+		if (this.payload.inlineAction.type === Constants.ActionSubmit && !this.isMultiline) {
```
[the remark this answers is not in the document]
Comment (Surezt22) [reply source/reactnative/src/components/inputs/input.js]: Done

**Pull request merged**